### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by Bigtee72** [keirf/greaseweazle#417] [Q&A]
This is my first discussion thanks to anyone who can help.
I have greaseweasle 4.1 all set up and working , all tracks copied to hard drive in scp format but don't work also files too small no where near 40mb.

**Comment by keirf** [comment]
More information is needed.

How big are the files? 20mb, 20kb, 100 bytes? It makes a big difference. Have you tried inspecting or converting the scp files, for example using techniques described in the recent Tech Tangents video https://www.youtube.com/watch?v=UxsRpMdmlGo&t=677s (hint: visualise your disk images using HxC software viewer).

You could tell us what you think is on the disks. What sort of drive you used. What command line you used (or what GUI and what boxes you ticked). You could zip and attach one of your scp image files. And so on.

**Reply by Bigtee72** [reply]
Thanks for getting back to me so quick , tried a few dd amiga cover disks new never used copy's all tracks but scp  file sizes range from kb to about 15mb. used gw in terminal also fluxmyfluffyfloppy . load files in amigafoever but don't work

**Reply by keirf** [reply]
Zip up and attach a couple here. The one that is a few kb, and one that is around 15mb. Then we can see if they contain anything useful.

**Comment by Bigtee72** [comment]
Thank you for your help ,here's a couple of  cover disks i've found sorry not games yet stuck in back of shed.
[game1.zip](https://github.com/keirf/greaseweazle/files/14549712/game1.zip)
[game2.zip](https://github.com/keirf/greaseweazle/files/14549713/game2.zip)

**Comment by keirf** [comment]
These dumps are terrible. There is nothing but garbage in game2. And in game1 it's mostly noise, but some tracks are partially decodeable on side 1. You can see this for yourself if you download HxC software (https://hxc2001.com/download/floppy_drive_emulator/HxCFloppyEmulator_soft_beta.zip) and load either of your SCP image files, and then click "Track Analyzer" and then "View mode: Dummy disk".

Reasons:
1. Bad or dirty drive. Is the drive known to read disks okay?
2. If it's a high-density drive, the density select switch could be dirty and non-functioning.

Either way, likely a drive problem. Get yourself a better drive. Also, bad dirty drives can even damage disks.

**Reply by Bigtee72** [reply]
Thanks for the help , The drive was from ebay he told me it was a good drive . its a mitsumi hd drive i cleaned it with cotton bud and alcohol so may be a bad drive. Do you recommend a certain make thanks

**Reply by keirf** [reply]
Yeah unfortunately drives can fail any time. Any PC drive will do, you just need a working one!

**Reply by keirf** [reply]
Also check your disks, if they're all from the same collection especially. Open the shutter and look for signs of mould or dirt. A mouldy disk won't read well and can damage a working drive.

**Reply by Bigtee72** [reply]
will order a new drive , the disk's i've been using are clean and new condition plastic wrapped . Thanks for all the help will let you know how i get on.

**Reply by Bigtee72** [reply]
Fitted new drive but having trouble writing to disc, jumper on but will not write. thanks for the help
[error.txt](https://github.com/keirf/greaseweazle/files/14672963/error.txt)

**Reply by keirf** [reply]
Written side 0 okay but not side 1. Can you read disks okay with this drive?

**Reply by Bigtee72** [reply]
It was reading disk's ok but not now, will take top off and clean heads just incase its picked something up . Thanks for the info

**Reply by keirf** [reply]
Yeah dirty disks might be a theme here

**Comment by Bigtee72** [comment]
Yes you might be right will give it a good clean tomorrow thanks for all your advice.On 20 Mar 2024 22:36, Keir Fraser ***@***.***> wrote:
Yeah dirty disks might be a theme here
